### ISSUE TITLE
Adapt namespace style to dotnet 6 style

### ISSUE THREAD

**Issue by LincolnTx** [rsaz/csharp-snippet-productivity#10]
Currently when i create a class it's start like this
```chsarp
using System;

namespace projectname.folder.subfolder
{

    public class NewOrderCommand
    {
    
    }
}
```

But the dotnet 6 styles has changed it and look so much cleaner, like this:

```
using System;

namespace projectname.folder.subfolder;

public class NewOrderCommand
{

}
```

Would be great if you can update this styles thanks!

**Comment by rsaz** [comment]
Hi @LincolnTx I will take a look on your request and implement it in the next release for sure. Thank you for the head's up!

**Comment by rsaz** [comment]
Hi @LincolnTx sorry for the late reply but you can find your request resolved in the 1.2.9 version. 
Now you can create .net 6.0 projects and create classes, interfaces, records, structs, with the new .net6.0 convention.

Thank you for reporting this and making the extension even better. Regards
